omero/users/cli/import.txt

```diff
@@ -54,7 +64,12 @@ Some of the options available to the import command are:
 
     Set the number of directories to scan down for files (default: 4)::
 
-        $ bin/omero import --depth 1 images_folder
+        $ bin/omero import --depth 7 images_folder
+
+    The above example changes the depth to 7 folders.
+
+Managing Performance of Imports
```

Comment:
lowercase to start words after the initial one please so this becomes "Managing performance of imports"